### PR TITLE
misc tutorial fixes

### PR DESCRIPTION
**Pull request by jpchen**
fixed formatting and english issues.

### CODE REVIEW

**Comment by neerajprad** [comment]
While you are at it, could you also help fix these warnings in our introductory tutorials `https://forum.pyro.ai/t/tensor-copy-construct-warning-in-svi-step/807`?

File: tutorial/source/intro_part_ii.ipynb [hunk 1]
```diff
@@ -246,32 +246,30 @@
...
+   "execution_count": 10,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "a =  0.6285385489463806\n",
-      "b =  9.107474327087402\n"
+      "a =  9.107474327087402\n",
```
Comment (neerajprad) [inline tutorial/source/intro_part_ii.ipynb]: I'm not able to see this rendered in github, but just a heads up that the output here seems to have been inter-changed. Is that okay?

Comment (jpchen) [reply tutorial/source/intro_part_ii.ipynb]: good catch!  all i did was rerun the cells. this seems to be another mistake, if you look at the [original graph](http://pyro.ai/examples/intro_part_ii.html#Next-Steps) `a` should be ~9.1 and `b` should be ~0.6. it must have been switched sometime and the cell wasnt rerun.

Comment (neerajprad) [reply tutorial/source/intro_part_ii.ipynb]: Good to know.

**Pull request merged**